terraform/modules/tf_classifier/iam.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_iam_role_policy" "classifier_policy" {
-  name   = "${var.function_name}_policy"
+  name   = "WriteAndEncryptSQS"
```

Comment:
no - `aws_iam_role_policy` resources get applied to the `aws_iam_role` itself. Namespacing here isn't require and actually hurts readability, since the _role_ itself **is** properly prefixed (now)

This PR is the prefixing fix PR.